filestore/s3.go

```diff
@@ -113,9 +108,13 @@ func (s *S3) Get(ctx context.Context, relPath string) (*File, io.ReadCloser, err
 		return nil, nil, err
 	}
 
+	size := int64(0)
+	if rsp.ContentLength != nil {
```

Comment:
Shouldn't an error be raised if it is nil?

Reply:
I honestly don't understand why they changed the field to be a pointer in the new versions. The docs also don't mention what a nil `ContentLength` mean. I think assuming it's 0 when nil is safe, but we can also be extra defensive.